std_msgs/String shape_type
std_msgs/String projection_group_name
std_msgs/String shape_id
---
std_msgs/Bool shape_removed